pkg/queue/breaker_test.go

```diff
@@ -27,9 +27,9 @@ import (
 )
 
 const (
-	// semAcquireTimeout is a timeout for tests that try to acquire
+	// semacquireTimeout is a timeout for tests that try to acquire
```

Comment:
I think your find/replace went a bit over the top on this one 🙂

Reply:
a bit overboard, indeed :)